dev/how-to/get-started/TiDB-Binlog-tutorial.md

```diff
@@ -0,0 +1,419 @@
+---
+title: 部署 TiDB-Binlog 
+summary: 在单个 TiDB 集群上部署 TiDB-Binlog 
+category: how-to
+---
+
+# 部署 TiDB-Binlog
+
+本文档将先介绍，在将数据迁移到 MariaDB 实例之前，如何在 Placement Driver、TiKV、TiDB、Pump 或 Drainer 各个组件中的单个节点上部署 TiDB-Binlog。
+
+希望上手实践 TiDB-Binlog 工具的用户需要对 [TiDB 架构]((https://pingcap.com/docs/architecture/))有一定的了解，最好有创建过 TiDB 集群的经验。阅读该文档也可以简单快速了解 TiDB-Binlog 架构。
+
+> **注意**：
+>
+> 该文档中部署 TiDB 的操作指导**不适用于**在生产或研发环境中部署 TiDB的情况。
+
+该文档假设用户使用的是现代 Linux 发行版本中的 x86-64。示例中使用的是 VMware 中安装运行的 CentOS 7 minimal 版本。强烈建议在一开始就进行清洁安装，以避免受现有环境中未知情况的影响。如果不想使用本地虚拟环境，你也可以使用云服务启动 CentOS 7 VM 版本。
+
+## TiDB-Binlog 介绍
+
+TiDB-Binlog 是一个用于收集 TiDB-Binlog 并提供实时数据备份和同步的工具。TiDB-Binlog 也可用于将 TiDB 集群的数据增量同步到下游。
+
+TiDB-Binlog 支持以下功能场景：
+- 数据增量备份：将 TiDB 集群中的数据复制到另一个集群；或通过 Kafka 发送 TiDB 更新数据并同步到下游。
+- 数据迁移：尤其是将数据从 TiDB 迁移到 MySQL/MariaDB。
```

Comment:
Please update the description according to the latest version.

Reply:
Updated.